rivery_cli/client.py

```diff
@@ -69,8 +71,11 @@ def __init__(self, **kwargs):
 
         self.token = kwargs.get('token')
         self.debug = kwargs.get('debug') or False
+        self.profile = kwargs.get('profile') or 'default'
         self.stack_path = kwargs.get('stack_path') or '.'
-        self.stack_name = kwargs.get('name') or kwargs.get('stack_name') or kwargs.get('client_name')
+        self.stack_name = kwargs.get('name') or kwargs.get('stack_name') \
+                          or kwargs.get('client_name') or self.profile
```

Comment:
what stack name is being used for? why is it the same as name, stack_name, client_name and profile?